database/migrations/000105_ruletype_evaluation_failure_message.up.sql

```diff
@@ -0,0 +1,17 @@
+-- Copyright 2024 Stacklok, Inc
+--
+-- Licensed under the Apache License, Version 2.0 (the "License");
+-- you may not use this file except in compliance with the License.
+-- You may obtain a copy of the License at
+--
+--      http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing, software
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+
+-- Adds a `evaluation_failure_message` column to the `rule_type` table. The failure message
+-- is displayed to the user when a rule evaluation fails.
+ALTER TABLE rule_type ADD COLUMN evaluation_failure_message TEXT NOT NULL DEFAULT '';
```

Comment:
*issue:* we're discussing an alternative proposal that does not need changes to the database. Let's sync before merging this.

Reply:
I confused "short message" with the longer description message, sorry for the noise.